pwnlib/tubes/os_process/process_windows.py

```diff
@@ -0,0 +1,960 @@
+import os
+import sys
+import time
+import ctypes
+import msvcrt
+import random
+import string
+import struct
+import socket
+import logging
+import threading
+
+from windows import *
+import windows
+import windows.winobject
+import windows.winproxy
+import windows.native_exec.nativeutils
+import windows.generated_def as gdef
+from windows.generated_def.winstructs import *
+import windows.native_exec.simple_x64 as x64
+import windows.debug
+
+try:
+    import capstone
+
+
+    def disasm(data, bitness=64, vma=0):
+        """disasm(data, bitness = 64, vma = 0) dissas the data at vma"""
+        cs = capstone.Cs(capstone.CS_ARCH_X86, capstone.CS_MODE_64 if bitness == 64 else capstone.CS_MODE_32)
+        dis = ''
+        for i in cs.disasm(data, vma):
+            dis += "%x:\t%s\t%s\n" % (i.address, i.mnemonic, i.op_str)
+        return dis
+except ImportError:
+    def disasm(data, bitness=64, vma=0):
+        raise (NotImplementedError("Capstone module not found"))
```

Comment:
Why are we doing this?  We do not use `capstone` as a disassembler, and importing it on `from pwn import *` greatly increases the startup time.

Do not overload / overwrite the names of other existing functions.  This should go in `asm.py`.